test/sql-tap/CMakeLists.txt

```diff
@@ -9,3 +9,104 @@ build_module(sql_uuid sql_uuid.c)
 target_link_libraries(sql_uuid msgpuck core)
 build_module(decimal decimal.c)
 target_link_libraries(decimal msgpuck)
+
+make_lua_path(LUA_PATH
+  PATHS
+    ${CMAKE_CURRENT_SOURCE_DIR}/lua/?.lua
+    ${PROJECT_SOURCE_DIR}/test/sql/lua/?.lua
+    ${PROJECT_SOURCE_DIR}/test/box/lua/?.lua
```

Comment:
Please add the comment why do we need "box" for sql tests.

Reply:
`test/box/?.lua` is needed for `identifier.lua` library. For example, in `sql-tap/identifier-characters.test.lua`.
Added a comment.

```diff
--- a/test/sql-tap/CMakeLists.txt
+++ b/test/sql-tap/CMakeLists.txt
@@ -14,6 +14,7 @@ make_lua_path(LUA_PATH
   PATHS
     ${CMAKE_CURRENT_SOURCE_DIR}/lua/?.lua
     ${PROJECT_SOURCE_DIR}/test/sql/lua/?.lua
+    # Needed for identifier.lua.
     ${PROJECT_SOURCE_DIR}/test/box/lua/?.lua
 )
 
```